repos/spack_repo/builtin/packages/py_numba/package.py

```diff
@@ -46,7 +47,7 @@ class PyNumba(PythonPackage):
     # max_numpy_run_version in setup.py is a non inclusive upper bound
     # min_numpy_run_version < min_numpy_build_version and these ranges use
     # min_numpy_build_version
-    depends_on("py-numpy@2.0:2.2", when="@0.61.2", type=("build", "run"))
+    depends_on("py-numpy@2.0:2.3", when="@0.61.2:", type=("build", "run"))
```

Comment:
numpy v2.3 only as of 0.62, https://github.com/numba/numba/commit/2f955dbc65784471ec888ae90d9b2697ed565a19
```suggestion
    depends_on("py-numpy@2.0:2.3", when="@0.62:", type=("build", "run"))
    depends_on("py-numpy@2.0:2.2", when="@0.61.2", type=("build", "run"))
```

Reply:
Thanks.